spec/models/manageiq/providers/openstack/cloud_manager/refresher_rhos_juno_spec.rb

```diff
@@ -35,4 +35,31 @@
       assert_with_skips
     end
   end
+
+  context "when paired with a infrastructure provider" do
```

Comment:
@rwsu rwsu, could you move this under infra_manager tests? @petrblaho  was preparing there a refresher test that will refresh both undercloud and overcloud, so all tests that will be testing that connection should go there